noxfile.py

```diff
@@ -15,17 +15,42 @@ def dev(session: nox.Session) -> None:
     development environment to ``.nox/dev``.
     """
     session.install("nox")
+    session.install("-e", ".[all]")
     session.install("-e", ".[tests]")
 
 
 @nox.session(python=["3.7", "3.8", "3.9"])
 def test(session: nox.Session) -> None:
     """Run unit tests."""
+    session.install("-e", ".[all]")
     session.install("-e", ".[tests]")
     session.run("airflow", "db", "init")
     session.run("pytest", *session.posargs)
```

Comment:
@tatiana so is the idea that we're running the example dags twice? Once with all deps and one with a subset of deps? this command will run those same examples.

Reply:
Exactly, @dimberman !
We still care about making sure things work when we have all the dependencies.